shell-completion/zsh/_coredumpctl

```diff
@@ -7,7 +7,7 @@ _coredumpctl_command(){
             'list:List available coredumps'
             'info:Show detailed information about one or more coredumps'
             'dump:Print coredump to stdout'
-            'gdb:Start gdb on a coredump'
+            'debug:Start debugger (gdb) on a coredump'
```

Comment:
Grammatical nitpick: this should be "a debugger" ("gdb" is a name, so no article, but "debugger" is a generic noun, so it needs one).